dipper/sources/MMRRC.py

```diff
@@ -383,10 +391,14 @@ def _process_phenotype_data(self, limit):
                         bkgd_id, genotype_id,
                         self.globaltt['unspecified_genomic_background'])
                     geno.addParts(
-                        gvc_id, genotype_id, self.globaltt['has_variant_part'])
+                        gvc_id, genotype_id, self.globaltt['has_variant_part'],
+                        part_category=blv.GenomicEntity.value,
+                        parent_category=blv.Genotype.value)
                     geno.addGenotype(genotype_id, genotype_label)
                     graph.addTriple(
-                        s, self.globaltt['has_genotype'], genotype_id)
+                        s, self.globaltt['has_genotype'], genotype_id,
```

Comment:
although it's not clear from the variable name, I think this should be categorized as an organism, (s = strain I think)

Reply:
Thanks @kshefchek! Fixed